forge/housekeeper/tasks/inviteReminder.js

```diff
@@ -0,0 +1,73 @@
+const { Op } = require('sequelize')
+
+const { randomInt } = require('../utils')
+
+module.exports = {
+    name: 'inviteReminder',
+    startup: false,
+    // runs every 15mins with a random offset from the start of the hour
+    schedule: `${randomInt(0, 15)}/15 * * * *`,
```

Comment:
```suggestion
    // runs daily at a randomly picked time
    schedule: `${randomInt(0, 59)} 6 * * *`,
```

Not sure why this needs to run so frequently? Once a day feels sufficient. Or am I missing a scenario that requires this to run more often?

Reply:
OK, so pinning it to a given hour means that the window to not run updates is a restart inside that hour.

Making it 6am is probably early enough that we are unlikely to do a code push/restart in that hour.